ui/desktop/CHANGELOG.md

```diff
@@ -2,9 +2,17 @@
 
 Canonical reference for changes, improvements, and bugfixes for Boundary Desktop.
 
+## V2.3.2 [CLI 0.19.2] (2025.TBD)
+
+### Bug Fixes
+
+- Fix target search field race condition. ([PR](https://github.com/hashicorp/boundary-ui/pull/2723))
+- Fix to conditionally display "Check for Updates" menu item. ([PR](https://github.com/hashicorp/boundary-ui/pull/2754))
```

Comment:
I might reword this to say something like "Removed check for updates when desktop client is installed through the boundary installer"

Reply:
great call, your suggestion sounds better, thank u!